### PR TITLE
Rs 2023.06.14

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 1]
```diff
@@ -462,12 +465,12 @@ Then... does that mean she derives sustenance from the same sorts of food that p
...
 -- Page 329, Offset 25085.
 -- 「なによ、いじわる」
-"What do you want, you bully?"
+"What do you want, meanie?"
 }
 [sha:574dd2276d3fa3c0eccbbc65b7728317f4fa1a7f]{
 -- Page 330, Offset 25086.
 -- 「いや、もう文句は言わないって。
-"Look, I won't complain any more.
+"Alright, alright, It's not about that.
```
Comment (Volvo-hackangel) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:473]: it's

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 2]
```diff
@@ -414,8 +416,9 @@ Even I don't understand why I'm acting like this.
 [sha:5e1b20b8ab27529a3029d111947c2bdeb71dfd91]{
 -- Page 325, Offset 25076.
 -- 「いらないっ。せっかく食事を摂る事にしたのに、そんな柔らかいものばっかりじゃモノを食べてるって気がしないもの」
-"I don't want them. I finally get a chance to eat something,
- but if I only eat soft things like those, it barely feels like I'm eating anything at all."
+"Keep em.
```
Comment (Volvo-hackangel) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:419]: nitpick but 'em for consistency :aokoheh:

File: script/Arcueid/Day 10/10_00_ARC10_5C.txt [hunk 3]
```diff
@@ -141,7 +141,7 @@ I had to push my way through the crowds in order to keep up with her.
 [sha:6826a5b160bdcd824198d6a9823880891187d860]{
 -- Page 512, Offset 25590. Glued.
 -- 人が多いのね。
- around at this time of night.
+ around at this time of night, aren't there.
```
Comment (Volvo-hackangel) [inline script/Arcueid/Day 10/10_00_ARC10_5C.txt:144]: question mark at the end instead?

File: script/Arcueid/Day 10/10_00_ARC10_5E.txt [hunk 4]
```diff
@@ -79,12 +79,12 @@ After I'm done with her, you're next.
...
 -- Page 631, Offset 25950.
 -- 「!?」
-"?!"
+"%{i}Hnh%{/i}?!"
 }
 [sha:37c979662d381f50050912144602ae5a8b2878a1]{
 -- Page 631, Offset 25951.
 -- 目の前に白い壁が立ち上がる。
-A white barrier forms before me.
+A white barrier springs up in front of me.
```
Comment (Petrikow) [inline script/Arcueid/Day 10/10_00_ARC10_5E.txt:87]: "A white screen" maybe. It needs to actually bring to mind a flat 'wall-like' structure.

Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_5E.txt:87]: 壁 gets translated as barrier quite a few times in this file, I guess change all?

Comment (Petrikow) [reply script/Arcueid/Day 10/10_00_ARC10_5E.txt:87]: sure, why not

**Pull request merged**